cfme/tests/cloud/test_cloud_events.py

```diff
@@ -40,7 +41,9 @@ def add_cmp(_, y):
                    event_type='networkSecurityGroups_write_EndRequest')
 
     def rm_cmp(_, y):
-        data = yaml.load(y)
+        # In 5.9 version `y` is a dict, not a yaml stream.
+        data = yaml.load(y) if provider.appliance.version < '5.9' else y
```

Comment:
**question** should we move the version dependent de-serialize to the framework side

the usage here seems limited, but i wonder if there are more call sites that need the behaviour